tests/lightning_qubit/test_measurements_class.py

```diff
@@ -788,6 +793,12 @@ def test_controlled_qubit_gates(self, operation, n_qubits, control_value, tol, l
                 tape = qml.tape.QuantumScript(ops, measurements)
 
                 statevector = lightning_sv(n_qubits)
+                if device_name == "lightning.tensor":
```

Comment:
can we combine these two ifs into one line

Reply:
Done!